src/components/alert/alert.stories.js

```diff
@@ -0,0 +1,255 @@
+import { html } from "lit-html";
+import "../button";
+import "../alert";
+import styles from "./styles.scss";
+import { stylesheet, docsIntro } from "../../../.storybook/utils.js";
+import { ALERT_SIZE, ALERT_TYPE, ALERT_ICON } from "./enums";
+
+export default {
+  title: "custom-elements/vl-alert",
+  decorators: [(story) => html`${stylesheet(styles)}${story()}`],
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          root: "alert",
+          intro:
+            "Use the alert (or notice) component to keep the user updated about important information.",
+        }),
+      },
+    },
+  },
+  args: {
+    title: "Opgelet!",
+    icon: "warning",
+    size: "normal",
+    type: "info",
+    closable: false,
+    buttonText: "Fout melden",
+    slotTitle: "Alert titel via slot",
+    content: "U heeft geen rechten om deze actie uit te voeren.",
+  },
+  argTypes: {
+    title: {
+      name: "data-vl-title",
```

Comment:
data-vl-title hoeft ook niet getoond worden in de with title slot story, want dat doet daar toch niets. Zoals bv hier: https://uig-webcomponents.omgeving.vlaanderen.be/?path=/story/custom-elements-vl-infoblock--with-slot-elements

Reply:
Done.